specification/dartLangSpec.tex

```diff
@@ -18227,14 +18230,20 @@ \section{Libraries and Scripts}
 
 \LMHash{}%
 The \Index{public namespace} of library $L$ is the namespace that maps
-the simple name of each public top-level member declaration $m$ of $L$ to $m$.
+the name of each public top-level member declaration $m$ of $L$ to $m$.
```

Comment:
"maps" suggests being a function (a name only maps to one declaration), but ....

Reply:
We have defined a (compile-time) namespace to be a partial function, so let's stick to that.